packages/victory-zoom-container/src/index.d.ts

```diff
@@ -10,20 +10,23 @@
 //                 Alec Flett <https://github.com/alecf>
 
 import * as React from "react";
-import { DomainPropType, VictoryContainerProps, CursorData } from "victory-core";
+import { RangeTuple, VictoryContainerProps, CursorData } from "victory-core";
 
 export interface VictoryZoomContainerProps extends VictoryContainerProps {
   allowPan?: boolean;
   allowZoom?: boolean;
-  clipContainerComponent?: React.ReactElement;
-  zoomDimension?: "x" | "y";
-  zoomDomain?: DomainPropType;
   brushStyle?: React.CSSProperties;
+  clipContainerComponent?: React.ReactElement;
   defaultBrushArea?: "all" | "none" | "disable";
   disable?: boolean;
   downsample?: number | boolean;
   minimumZoom?: CursorData;
```

Comment:
`CursorData` is a really confusingly named variable. I know you didn't name it, but do you mind changing it?

Reply:
Sure I can do that